src/main/java/org/jenkinsci/plugins/pipeline/github/PullRequestGroovyObject.java

```diff
@@ -137,6 +137,11 @@ public int getMilestone() {
         return pullRequest.getMilestone().getNumber();
     }
 
+    @Whitelisted
+    public String getMilestoneTitle() {
```

Comment:
I think this is good, but I wonder if we should perhaps return a `Milestone` object?

https://developer.github.com/v3/issues/milestones/

Reply:
@aaronjwhiteside I'd be happy to make this adjustment. I have a working version locally, but was wondering how to handle this change potentially breaking existing usages of `getMilestone` since that currently returns an `int` representing the milestone number.

Reply:
I'm happy to bump the version to indicate that there is a breaking API change... 

Alternatively we could always add an additional method to return the object, `getMilestone2()` (I'm not sold on that name either), and deprecated the existing `getMilestone()` method, citing that it will eventually return a `MilestoneGroovyObject`.. and to use the alt method in the meantime?

Reply:
@aaronjwhiteside Sounds good. I just pushed up my changes with the updated Milestone object. I've also updated the README, but haven't adjusted the version number. I'd prefer your suggest for just updating the version as a breaking API change. Thanks for getting back and sorry for the delays on this one!